cascades-server/configuration/src/test/java/pl/gov/coi/cascades/server/presentation/PresentationConfigurationTest.java

```diff
@@ -66,23 +63,27 @@ public void testProduceDatabaseNameGeneratorService() throws Exception {
     }
 
     @Test
-    public void testProduceCredentials() throws Exception {
-        // given
-        PresentationConfiguration presentationConfiguration = new PresentationConfiguration();
-
+    public void testProduceCredentials() {
         // when
-        UsernameAndPasswordCredentialsGeneratorService actual = presentationConfiguration.produceCredentials();
+        UsernameAndPasswordCredentialsGeneratorService actual = presentationConfiguration.produceCredentials(new SecureRandom());
 
         // then
         assertThat(actual).isNotNull();
         assertThat(actual).isInstanceOf(UsernameAndPasswordCredentialsGeneratorService.class);
     }
 
     @Test
-    public void testProduceDatabaseTypeClassName() throws Exception {
-        // given
-        PresentationConfiguration presentationConfiguration = new PresentationConfiguration();
+    public void testProduceRandomGenerator() {
+        // when
+        Random actual = presentationConfiguration.produceRandomGenerator();
+
+        // then
+        assertThat(actual).isNotNull();
+        assertThat(actual).isInstanceOf(ThreadLocalRandom.class);
```

Comment:
There should be tests to assert for implementation, only for contract. In this case: `Random`. `ThreadLocalRandom` should not be concern here.